src/book/summary.rs

```diff
@@ -101,11 +100,48 @@ impl Default for Link {
     }
 }
 
-/// An item in `SUMMARY.md` which could be either a separator or a `Link`.
+/// A linked virtual chapter in the `SUMMARY.md`, possibly with nested entries.
+///
+/// This is roughly the equivalent of `- Some virtual section`.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct VirtualLink {
```

Comment:
What would the difference be between a normal link and a virtual one? My understanding is you'd write a normal link as `- [Some Chapter](./chapter.md)` and a virtual link as `- [Some Chapter]()` (i.e. virtual chapters have no link in `SUMMARY.md`).

Reply:
I think, that is an important design decision to make!

I propose that a user can write

- `- [Some Chapter](./chapter.md)` for a link and
- both `- Some virtual chapter` and `- [Some virtual chapter]()` for a virtual link.

Maybe, we could add a lint for `- [Some virtual chapter]()` but I would rather not do that.

Reply:
For the moment, I've only implemented parsing `- [Some virtual chapter]()`. `- Some virtual chapter` seems to be more tricky, therefore I would like to have a decision before going on... ;-)